src/System.Private.ServiceModel/tests/System.Private.ServiceModel.Tests.csproj

```diff
@@ -14,7 +14,7 @@
     <ProjectGuid>{BB3B114D-68D7-4E36-9EBE-127E694E6EC9}</ProjectGuid>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs" Exclude="Scenarios\SelfHostWcfService\*" />
+    <Compile Include="**\*.cs" Exclude="Scenarios\**\*" />
```

Comment:
+1. This issue was causing a lot of grief, and I had removed it from any local repo I had, but didn't submit it for PR. Also helps with load performance.